tensorflow/python/compiler/tensorrt/trt_convert.py

```diff
@@ -159,12 +167,14 @@ class TrtConversionParams(
   """
 
   def __new__(cls,
-              max_workspace_size_bytes=DEFAULT_TRT_MAX_WORKSPACE_SIZE_BYTES,
+              max_workspace_size_bytes=None,
               precision_mode=TrtPrecisionMode.FP32,
               minimum_segment_size=3,
               maximum_cached_engines=1,
               use_calibration=True,
               allow_build_at_runtime=True):
+    if max_workspace_size_bytes is None:
+      max_workspace_size_bytes = _get_default_max_workspace_size()
```

Comment:
we can either protect the call here with _pywrap_py_utils.is_tensorrt_enabled() or add such a check to the callee implementation

Reply:
Thanks for the tip, I brought back the constant and guarded the call with `_pywrap_py_utils.is_tensorrt_enabled()`

Reply:
@bixia1 `_pywrap_py_utils` uses a `LazyLoader`, using it at import time will import the module, which could have some side effects. I'm not sure why a lazy loader is used here, I'll ask the team.

Reply:
It seems `_pywrap_py_utils` isn't built in CPU-only builds.

Reply:
@reedwm It seems you added the line where `_pywrap_py_utils` is lazy-loaded, perhaps you remember if there was any reason not to want this module to be loaded when `trt_convert` is?

Reply:
The commit I authored, 2e7bd979e18fd6745255c798e73e058a7f4c5989 didn't switch `_pywrap_py_utils` to be lazy-loaded. It just switched it to use pybind11.

2e6d84b216cc36a7f8d39abf756d270c1bf5e2ba switched it to be lazy-loaded, with the commit message

> Lazily load the wrap_py_utils module so that python binaries that doesn't have access to the wrapped dso won't crash at startup.

I suppose this means not all binaries have access to the `_pywrap_py_utils` object, but I'm not sure. Perhaps `DEFAULT_TRT_CONVERSION_PARAMS` itself should be lazily initialized by having a function, `get_default_trt_conversion_params()`, that sets `DEFAULT_TRT_CONVERSION_PARAMS` if it hasn't already been set.

Reply:
@reedwm Thanks for your answer. Lazy initialization of `DEFAULT_TRT_CONVERSION_PARAMS` (or simply replacing it with `get_default_trt_conversion_params()`) could be a solution, but likely to break external codebases that import `trt_convert`, get the parameters, modify them and invoke TF-TRT. A last-resort solution.

@aaroey By "wrapped dso", did you mean `libnvinfer.so` or a TF dso? The module can be loaded successfully without the former.